packages/fast-components-react-base/src/tree-view-item/tree-view-item.props.ts

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { ManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { TreeViewItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface TreeViewItemManagedClasses
+    extends ManagedClasses<TreeViewItemClassNameContract> {}
+
+export interface TreeViewItemUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface TreeViewItemHandledProps extends TreeViewItemManagedClasses {
+    /**
+     * Renders the items expand/collapse glyph
+     */
+    expandCollapseGlyph?: (className?: string) => React.ReactNode;
+
+    /**
+     * The expanded state of the tree view item on mount.
+     * After mounting, the tree view item will track the expanded
+     * state internally to align to accessibility guidelines.
+     * This property will have no effect if the item has no child nodes
+     */
+    defaultExpanded?: boolean;
```

Comment:
Do we need an expanded prop for controlled mode?

Reply:
This is technically the `expanded` property, so we could likely just call it `expanded` instead of default expanded.

Reply:
@scomea, after sleeping on this - I'm not sure that `expanded` would be *as* accurate for this control as `defaultExpanded`. Essentially, the expanded state is not *explicitly* controlled here. 

I could set expanded on a `treeitem`, but if it does not have childNodes, it would not (nor do I think it should) actually behave as expanded. If children were dynamically added after the initial render, it would default to the `defaultExpanded` prop.

Similarly, if I wanted to default a given parent node to NOT be expanded, but a child node within the tree were `selected` on mount, the parent node(s) above that child should expand for accessibility concerns. 

With the above scenarios, I feel like `defaultExpanded` provides a better API as it is more informative of the actual behavior of the tree. We'll honor that so far as we can given the accessible requirements. As a user, I would expect `expanded` to be more definitive, whereas `defaultExpanded` seems to convey the potential for it to *not be the case*.